### ISSUE TITLE
Consider a non-Google-based captcha

### ISSUE THREAD

**Issue by ryliejamesthomas** [neocities/neocities#323]
Hi! I love neocities, but really hate Google. Just wondering if you'd consider an alternative to Google's recaptcha, so I don't have to enable them in my blocking rules.

There are some alternatives on this page: https://switching.software/ethical-alternatives-to-google-recaptcha/

- [Captcheck](https://captcheck.netsyms.com/)
- [Click Captcha](https://github.com/Lokno/click-captcha)
- [Securimage](https://www.phpcaptcha.org/)

Another I've heard good things about:

- [kocaptcha](https://github.com/koto-bank/kocaptcha)

**Comment by stranger195** [comment]
Any updates or thoughts on this? @kyledrake

**Comment by joppiesaus** [comment]
[hcaptcha](https://www.hcaptcha.com/) seems robust

**Comment by rnbunker** [comment]
@kyledrake

**Comment by kyledrake** [comment]
Agreed 100%, hcaptcha looks great. I've been using it for the contact form as a test before moving the main signup over. Working on it, just need to be careful to implement it correctly because it's important.

**Comment by airone506** [comment]
This is awesome! Thank you so much for this.
